src/invidious/search.cr

```diff
@@ -1,12 +1,18 @@
+class ChannelSearchException < InfoException
+  def initialize(channel : String)
+    super "Unable to find channel with id of '#{channel}'. Are you sure that's an actual channel id?"
```

Comment:
It would be preferrable to have a generic exception in here, then to properly catch it and use translation strings for the error in the route handler.

Reply:
By generic, I'm thinking you mean to just keep the `channel` as an instance variable and then prepare this message in the route handler. But if you mean that I should use `InfoException`, I'd worry that it would be errors this PR was not meant to handle

Reply:
I was thinking about an exception without a message (i.e `class Foo < InfoException; end`) with the rescuing and error message done in the route handler.

PS: after looking at the code, the error template doesn't allow for a parametric string anyway.